### DISCUSSION TITLE
[求助]基尼奇专武都谁可用

### DISCUSSION THREAD

**Discussion by niranzhu** [share121/inter-knot#4838] [原神]
早上中午晚上好各位旅行者们！
我最近有抽到基尼奇的专武的臆想，但不太清楚除了基尼奇自己之外，还有哪些角色可以使用这把专武呢？
山王长牙的白值看起来很强大，但特效难以叠满。我想充分发挥这把专武的价值，让它在其他角色手上也能绽放光彩。
希望各位有研究的大佬们能给我一些建议，感激不尽！
![37ec70a8fb76dd117cef55880214022cf5edc1a2](https://github.com/user-attachments/assets/86bd7a31-1896-47f9-8e5a-1b424f7bd91f)

**Comment by Junjun280** [comment]
求封面图嘿嘿
